app/client/models/entities/PageRec.ts

```diff
@@ -39,4 +44,20 @@ export function createPageRec(this: PageRec, docModel: DocModel): void {
     return this.isCensored() || this.isSpecial();
   });
   this.share = refRecord(docModel.shares, this.shareRef);
+  const options = modelUtil.jsonObservable(
+    this.options,
+    (obj: any) => obj || {}
+  );
+  const collapsed = modelUtil.fieldWithDefault(
+    options.prop("collapsed"),
```

Comment:
The `prop()` machinery is kind of old and unnecessary. You can achieve the same without it, e.g.:
```
// To read
Computed.create(this, use => Boolean(use(options).collapsed));

this.isCollapsed = Observable.create(this, this.isCollapsedByDefault.get());

// To save
await options.setAndSave({...options.peek(), collapsed: value});
```